podman_compose.py

```diff
@@ -984,17 +969,17 @@ def get_net_args_from_networks(compose, cnt):
                 )
 
     for net_, net_config_ in multiple_nets.items():
-        net_desc = compose.networks[net_] or {}
+        net_desc = compose.networks.get(net_) or {}
         is_ext = net_desc.get("external")
         ext_desc = is_ext if isinstance(is_ext, str) else {}
         default_net_name = default_network_name_for_project(compose, net_, is_ext)
         net_name = ext_desc.get("name") or net_desc.get("name") or default_net_name
 
         ipv4 = net_config_.get("ipv4_address")
         ipv6 = net_config_.get("ipv6_address")
-        # custom extension; not supported by docker-compose v3
+
         mac = net_config_.get("x-podman.mac_address")
-        aliases_on_net = norm_as_list(net_config_.get("aliases", []))
+        aliases_on_net = net_config_.get("aliases")
```

Comment:
Probably makes sense to move to separate commit, as it's yet another cleanup.

Reply:
Just rollback changes here.